packages/frontend/pages/gdt.vue

```diff
@@ -0,0 +1,174 @@
+<!-- gdt.vue
+
+© 2024 Global Open Source Quality Assurance System. All rights reserved.
+We are committed to keeping our code open source, but all GOSQAS and GDT 
+branding, including logos, is subject to the copyright above.
+
+Copyright (C) 2024 GOSQAS
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>. -->
+<!--
+    This is the landing page where you can create a new record to track
+-->
+<template>
+    <link href="https://fonts.google.com/specimen/Poppins" rel="stylesheet" type="text/css">
```

Comment:
Does this successfully load for you? I've always got a MIME type warning when getting font but I haven't looked into it.

Reply:
Not sure what that line was doing as the font style comes from elsewhere, I just removed it as it seems to have no effect.